pkg/forward/forward.go

```diff
@@ -447,9 +447,8 @@ func (isdf *InterStepDataForward) streamMessage(
 		// to send the result to. Then update the toBuffer(s) with writeMessage.
 		msgIndex := 0
 		for writeMessage := range writeMessageCh {
-			// add partition to the ID, to make sure that the ID is unique across partitions
 			// also add vertex name to the ID, since multiple vertices can publish to the same vertex and we need uniqueness across them
```

Comment:
this comment now sounds a little weird starting with "also" given that the previous comment was removed - maybe the comments here can say what each of the components are? (same issue on line 672 below)

Reply:
done